20200803 checker

changlong-liu opened the pull request.

LianwMS commented on src/utils/helper.ts:
```diff
@@ -315,4 +315,4 @@ export function parseResourceId(mpath: string): Map<string, string> {
         ret.set("last_child_num", "" + count);
     }
     return ret;
-}
```
undo change since no actually changes

LianwMS commented on src/plugins/azgenerator/TemplateAzureCliTestScenario.ts:
```diff
@@ -48,13 +47,13 @@ export function GenerateAzureCliTestScenario(model: CodeModelAz): string[] {
+    //     if (exampleId) {
+    //         model.FindExampleById(exampleId, commandParams, []);
+    //     }
+    // }
```
Remove if no needed

LianwMS commented on src/plugins/azgenerator/ScenarioTool.ts:
```diff
@@ -201,41 +200,169 @@ class ResourceClass {
 }
+
+export enum ObjectStatus {
+    NA,
```
>NA [](start = 4, length = 2)

None?

LianwMS commented on src/plugins/azgenerator/TemplateAzureCliTestScenario.ts:
```diff
@@ -101,7 +103,21 @@ export function GenerateAzureCliTestScenario(model: CodeModelAz): string[] {
                     }
-                    steps.push("    " + disabled + "         checks=[])");
+                    let checks =  model.GetExampleChecks(examples[exampleIdx++]);
+                    if (checks.length>0) {
```
>if (checks.length>0 [](start = 20, length = 19)

format the code here

qiaozha commented:
Hold on this PR until we confirmed with CLI guys.

qiaozha commented:
@changlong-liu  Cam we add the not-generate check option in the e2e test, otherwise I guess the e2e test we added will fail.

changlong-liu commented:
> @changlong-liu Changlong Liu FTE Cam we add the not-generate check option in the e2e test, otherwise I guess the e2e test we added will fail.

Hi @qiaozha , added the e2e PR: https://github.com/amecodegenbot/azure-rest-api-specs/pull/6
